dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/DynamicVirtualHost.java

```diff
@@ -479,6 +482,9 @@ private boolean startWebApp(WebApp webApp) {
     }
 
     public boolean stopWebApplication(DeployedModule dm) {
+        if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)){
+            logger.logp(Level.FINE, CLASS_NAME,"stopWebApplication",  "Enter ");
+        }
```

Comment:
Include the String ct in the Enter log, so at least you know what context is stopping

Reply:
we will not have right string until it passes next line in makepropercr so , lets not print it, since it can be diff.